src/grid.js

```diff
@@ -119,15 +120,25 @@ export const getLevelByViewport = (boxRange, viewportRange) => {
  * @param {number} width 
  * @param {number} height 
  * @param {number} boxRange 
- * @param {Array.<number, number>} center [lon, lat]
+ * @param {Object} center {lon, lat}
+ * @param {bool} fixIntegrity Ensure that computed extent will not overfloat antimeridian. Example: longitude -190 is changed to -180.
+ * @param {Object} center {lon, lat}
  */
 export const getTileGrid = (width, height, boxRange, center, fixIntegrity) => {
     const viewportRange = mapUtils.view.getMapViewportRange(width, height);
-    const zoom = mapUtils.view.getZoomLevelFromBoxRange(boxRange, width, height);
-    const boxRangeByZoomLevel = mapUtils.view.getBoxRangeFromZoomLevel(zoom, width, height);
     const level = getLevelByViewport(boxRange, viewportRange);
     const ratio =  width / height;
-    const extent = getExtentAroundCoordinates(center, boxRangeByZoomLevel, ratio, 50, fixIntegrity);
-    const tileGrid = getGridForLevelAndExtent(level, extent, fixIntegrity);
+    const extent = mapUtils.view.getBoundingBoxFromViewForEpsg3857(
```

Comment:
What if we want an extent for a map in another projection?

Reply:
Good point.
I dont know if we should deal it now.
There is few options.
1) rename this function to reflect usability for epsg:3857 
2) let it go and deal it as we need
...

Reply:
I'm for option 2, because we can't handle extent for 3D accurrately right now . Maybe we can return whole world by default. And, please, add todo.

Reply:
done: https://github.com/gisat-panther/ptr-tile-grid/pull/14/commits/47963076b13771b9af8c814bbbff6bb8f4ac555d